### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by mj-e-boc**
-Add index.html for menu
-Add stylesheet for menu
-Implement behaviour for menu
-Fix linter errors for menu

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+function hideMenu() {
+  navMenu.style.left = '0';
+}
+
+hamburger.addEventListener('click', hideMenu);
+
+document.querySelectorAll('.navMenu li').forEach((n) => n.addEventListener('click', () => {
+  navMenu.style.left = '200%';
+}));
```
Comment (GzimAsani) [inline index.js:19]: - [ ] When first loading the project the menu needs to be closed, currently it is opened when first starting your app

File: index.html [hunk 2]
```diff
@@ -53,6 +53,23 @@
...
+          <a href="#portfolio"> Portfolio</a>
+        </li>
+        <li class="menuItem">
+          <a href="#aboutme">About</a>
+        </li>
+        <li class="menuItem">
+          <a href="#contactme">Contact</a>
+        </li>
+      </ul>
+    </div>
```
Comment (GzimAsani) [inline index.html:72]: - [ ] Give it some padding bottom in order to match the design better 
image

File: index.js (hunk 1 above)
Comment (GzimAsani) [reply index.js:19]: - [ ] This does not need to be present on desktop 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @mj-e-boc                        :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with implementing the menu, it looks great +1: 

 ## Required Changes 

- [ ] Check for the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @mj-e-boc 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] KIndly make sure to center the header contents which will increase your portfolio to look better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**